ci/steps.yml

```
@@ -243,9 +243,15 @@ steps:
     } else {
       Write-Host '##vso[task.setvariable variable=ARCHITECTURE]x86'
     }
-    if ('$(ARCH)' -like '*msvc*') {
+    if ('$(ARCH)' -like '*msvc2022*') {
       Write-Host '##vso[task.setvariable variable=VSVER]2022'
     }
+    if ('$(ARCH)' -like '*msvc2019*') {
+      Write-Host '##vso[task.setvariable variable=VSVER]2019'
+    }
+    if ('$(ARCH)' -like '"*_arm64') {
+      Write-Host '##vso[task.setvariable variable=ARCHITECTURE]arm64'
```

```suggestion
      Write-Host '##vso[task.setvariable variable=ARCHITECTURE]amd64_arm64'
```
You would only use `arm64` here if the host were ARM64 too, it's different [when cross-compiling](https://learn.microsoft.com/en-us/cpp/build/building-on-the-command-line?view=msvc-170#vcvarsall-syntax).